tests/app/main/test_briefs.py

```diff
@@ -57,6 +56,41 @@
     ' could provide this specialist role when you applied to the Digital Outcomes and Specialists framework.'
```

Comment:
The below is a *Too complex* error. No need to define the class within the method